lib/ddtrace/contrib/action_view/utils.rb

```diff
@@ -23,6 +23,7 @@ def normalize_template_name(name)
           else
             sections_view[-1]
           end
+        # rubocop:disable Lint/RescueWithoutErrorClass
```

Comment:
I'm in favour of ignoring `Lint/RescueWithoutErrorClass`. `StandardError` being the default is not really a well-kept secret.

Reply:
It's a good point. Rubocop is always a bit of a love/hate -- their defaults are awful and the cops rarely have a good through discussion of what we should be on the look out for.

This to say that I do think there can be some value to this suggestion, which is to push people to think if they are writing a catch-all "I need this to never fail", or if they're trying to act on a specific error in a specific case and "forgot" to be specific.

So... I'll leave it for now, but I won't overly mind if in the future we end up tweaking that default, if that's reasonable :)